tools/souper.cpp

```diff
@@ -44,24 +44,6 @@ OutputFilename("o", cl::desc("Override output filename"),
 static cl::opt<bool> StaticProfile("souper-static-profile", cl::init(false),
     cl::desc("Static profiling of Souper optimizations (default=false)"));
 
-static cl::opt<bool> PrintNeg("print-neg", cl::init(false),
```

Comment:
hmm, what were these options doing in the code? @jubitaneja ?

Reply:
These are dead codes

Reply:
yeah I was wondering how they got in there in the first place, so we can do a better job avoiding nasty stuff like this in the future